postfix/assets/configuration/spec.yaml

```
@@ -36,8 +36,10 @@ files:
   - template: instances
     options:
     - name: directory
-      required: true
-      description: Path to the postfix directory.
+      required: false
+      description: |
+        Path to the postfix directory. The directory parameter is required if `postqueue: false` is set. For more 
```

small nit, for consistency
```suggestion
        Path to the postfix directory. The directory option is required if `postqueue: false` is set. For more 
```